### ISSUE TITLE
Config loading for both classpath & URL

### ISSUE THREAD

**Issue by clgy15** [geomesa/geomesa-nifi#7]
Geomesa-Convert already includes an implementation to load converters & simple feature types from a URL instead of searching on the classpath or in config files. These are defined in ConverterConfigLoader.scala

Currently on the Nifi processors, it uses the ConverterConfigLoader/SimpleFeatureTypeLoader to find valid configs. Changing them to use URL loaders is not difficult, however the trick here is to allow people to select between a classpath or a URL in Nifi. @jahhulbert-ccri and I talked about a controller service to allow where things like the location of the configs are. 

Unfortunately I lost my implementation for the processor switch, but it was manual and didn't allow one to switch between the two..

tl;dr Include a way for our nifi processors that use simplefeature/converter configs to use either a classpath or a URL as a valid point to pull them from.

**Comment by elahrvivaz** [comment]
You should be able to do this by providing a jar with an application.conf that contains `geomesa.convert.config.urls: https://foo.bar`, and adding that jar to the ExtraClasspaths processor configuration.
